<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAuditReports</class>
 <widget class="QWidget" name="FormAuditReports">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Générateur de rapports d'audit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Images.qrc">
    <normaloff>:/icons/icons/report.png</normaloff>:/icons/icons/report.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButtonCollapseAll">
         <property name="text">
          <string>Replier tout</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QComboBox" name="comboListAudits"/>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboListTypes"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTreeView" name="treeView">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Clics gauches pour sélectionner les noeuds à inclure dans le rapport, double clic pour sélectionner le noeud de référence pour les comparaisons de certains tableaux et graphiques.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>false</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButtonExpandAll">
         <property name="text">
          <string>Déplier tout</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButtonSelectType">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sélection par type&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Sélectionner les :</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButtonSelectAllNodes">
         <property name="text">
          <string>Sélectionner tout</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButtonUnSelectAllNodes">
         <property name="text">
          <string>Sélectionner rien</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget2">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBoxDF">
         <property name="text">
          <string>Dépenses de fonctionnement</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkBoxRF">
         <property name="text">
          <string>Recettes de fonctionnement</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBoxDI">
         <property name="text">
          <string>Dépenses d'investissement</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="checkBoxRI">
         <property name="text">
          <string>Recettes d'investissement</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QListWidget" name="listTables">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>3</verstretch>
          </sizepolicy>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Liste des tableaux pouvant être incorporés dans le rapport. Seul les éléments sélectionnés seront pris en compte.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QListWidget" name="listGraphics">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>2</verstretch>
          </sizepolicy>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Liste des graphiques pouvant être incorporés dans le rapport. Seul les éléments sélectionnés seront pris en compte.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Liste de préselections de tableaux et de graphiques. Un clic sur un des boutons ajoute la préselection à la sélection de tableaux et de graphiques actuelle.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="title">
          <string>Présélections</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButtonJoursAct">
            <property name="text">
             <string>Jours activité</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButtonPoidsRelatifs">
            <property name="text">
             <string>Poids relatifs</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonEvolutionCumul">
            <property name="text">
             <string>Évolution cumulée</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButtonEvolution">
            <property name="text">
             <string>Évolution</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButtonBase100">
            <property name="text">
             <string>Base 100</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="pushButtonSelectNone">
            <property name="text">
             <string>Rien</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="pushButtonSelectAll">
            <property name="text">
             <string>Tout</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButtonResultats">
            <property name="text">
             <string>Résultats</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Tableaux</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Graphiques</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxTraversal">
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ordre dans lequel les noeuds seront présentés à l'intérieur du rapport.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="title">
      <string>Ordre d'affichage des noeuds</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioButtonDFS">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Si l'arbre est :&lt;/p&gt;&lt;p&gt;R&lt;/p&gt;&lt;p&gt;----A&lt;/p&gt;&lt;p&gt;---------A1&lt;/p&gt;&lt;p&gt;---------A2&lt;/p&gt;&lt;p&gt;----B&lt;/p&gt;&lt;p&gt;---------B1&lt;/p&gt;&lt;p&gt;---------B2&lt;/p&gt;&lt;p&gt;Le rapport contiendra dans cet ordre :&lt;/p&gt;&lt;p&gt;R, A, A1, A2, B, B1, B2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Profondeur d'abord</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonBFS">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Si l'arbre est :&lt;/p&gt;&lt;p&gt;R&lt;/p&gt;&lt;p&gt;----A&lt;/p&gt;&lt;p&gt;---------A1&lt;/p&gt;&lt;p&gt;---------A2&lt;/p&gt;&lt;p&gt;----B&lt;/p&gt;&lt;p&gt;---------B1&lt;/p&gt;&lt;p&gt;---------B2&lt;/p&gt;&lt;p&gt;Le rapport contiendra dans cet ordre :&lt;/p&gt;&lt;p&gt;R, A, B, A1, A2, B1, B2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Largeur d'abord</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="saveButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>47</height>
      </size>
     </property>
     <property name="text">
      <string>Générer le rapport</string>
     </property>
     <property name="icon">
      <iconset resource="Images.qrc">
       <normaloff>:/icons/icons/html.png</normaloff>:/icons/icons/html.png</iconset>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboListAudits</tabstop>
  <tabstop>treeView</tabstop>
  <tabstop>pushButtonExpandAll</tabstop>
  <tabstop>pushButtonCollapseAll</tabstop>
  <tabstop>pushButtonSelectAllNodes</tabstop>
  <tabstop>pushButtonUnSelectAllNodes</tabstop>
  <tabstop>pushButtonSelectType</tabstop>
  <tabstop>comboListTypes</tabstop>
  <tabstop>radioButtonDFS</tabstop>
  <tabstop>radioButtonBFS</tabstop>
  <tabstop>checkBoxDF</tabstop>
  <tabstop>checkBoxRF</tabstop>
  <tabstop>checkBoxDI</tabstop>
  <tabstop>checkBoxRI</tabstop>
  <tabstop>listTables</tabstop>
  <tabstop>listGraphics</tabstop>
  <tabstop>pushButtonPoidsRelatifs</tabstop>
  <tabstop>pushButtonEvolution</tabstop>
  <tabstop>pushButtonBase100</tabstop>
  <tabstop>pushButtonSelectAll</tabstop>
  <tabstop>pushButtonJoursAct</tabstop>
  <tabstop>pushButtonEvolutionCumul</tabstop>
  <tabstop>pushButtonResultats</tabstop>
  <tabstop>pushButtonSelectNone</tabstop>
  <tabstop>saveButton</tabstop>
 </tabstops>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
